deployments/modules/container_build_push/main.tf

```diff
@@ -25,7 +25,7 @@ locals {
   build_args_str = join(" ", [for arg in var.build_args : "--build-arg ${arg}"])
 }
 
-resource "terraform_data" "image" {
+resource "terraform_data" "image_inputs" {
```

Comment:
we should also update container_binary_upload main.tf to use the same terraform_data pattern